cypress/integration/Home_spec.js

```diff
@@ -0,0 +1,106 @@
+beforeEach(() => {
+  cy.visit('http://localhost:3000');
+});
+
+describe('App main display', () => {
+  it('Should confirm true is equal to true', () => {
+    expect(true).to.equal(true)
+  });
+
+  // it('Clicking on a link should take user to books', () => {
+  //   cy.get('.home-page-btn').click({ multiple: true })
+  //   cy.get('.book-card-grid').should('be.visible')
+  //   cy.get('.book-cover-img').should('be.visible')
+  //   // cy.contains('https://covers.openlibrary.org/b/id/7424001-L.jpg')
+  //   cy.location().should((loc) => {
+  //     expect(loc.href).to.eq('http://localhost:3000/books')
+  //   })
+  // });
+
+
+  it('Clicking on biography should take user to books', () => {
```

Comment:
Nice! Here is an intercept example I used that might help in this block of tests
```
cy.intercept('GET', 'https://openlibrary.org/subjects/biography.json', {
      statusCode: 201,
      body: {
        name: 'biography',
        works: [
          {title: 'Brown Bear', cover_id: 1, key: '/works/OL503666W', authors:[{name: 'Nico'}]},
          {title: 'Hungry Caterpillar', cover_id: 2, key: '/works/OL362702W', authors:[{name: 'Kody'}]},
          {title: 'Cat in the Hat', cover_id: 3, key: '/works/OL107195W', authors:[{name: 'Renee'}]}
        ]
      }
    })
```